app/views/layouts/application.html.erb

```diff
@@ -11,10 +11,18 @@
     <div class="container-fluid">
       <div class="row">
         <div class="col-10">
-          <!--To be cleaned up so they don't disrupt flow of made (bootstrap modal maybe) -->
-          <p class="notice"><%= notice %></p>
-
-          <p class="alert"><%= alert %></p>
+          <!--To be cleaned up so they don't disrupt flow of page (bootstrap modal maybe) -->
```

Comment:
You could also extract this into a view partial called `_flashes.html.erb`. If you do this, create a shared folder under views: `app/views/shared/_flashes.html.erb` and include it like this: `render "shared/flashes"`.